src/libostree/ostree-repo-static-delta-compilation.c

```diff
@@ -636,7 +636,7 @@ try_content_rollsum (OstreeRepo                       *repo,
   if (!get_unpacked_unlinked_content (repo, to, &tmp_to, cancellable, error))
     return FALSE;
 
-  OstreeRollsumMatches *matches = _ostree_compute_rollsum_matches (tmp_from, tmp_to);
+  g_autofree OstreeRollsumMatches *matches = _ostree_compute_rollsum_matches (tmp_from, tmp_to);
```

Comment:
Shouldn't this be

```suggestion
  g_autoptr(OstreeRollsumMatches) matches = _ostree_compute_rollsum_matches (tmp_from, tmp_to);
```

?

Reply:
Yep done